maat-court-data-api/src/main/java/gov/uk/courtdata/entity/ChildWeightHistoryEntity.java

```diff
@@ -0,0 +1,46 @@
+package gov.uk.courtdata.entity;
+
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
```

Comment:
this '@Data' annotation covers most annotation like getter and setter etc.

Reply:
Done